apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -869,23 +869,25 @@ When set to true, invalid utf8 strings in for example client ID, topic name, etc
 
     mqtt_max_awaiting_rel {
         desc {
-            en: """Maximum QoS 2 packets (Client -> Broker) awaiting PUBREL."""
-            zh: """PUBREL (Client -> Broker) 最大等待队列长度。"""
+            en: """EMQX creates a wait queue in the client's session to accept QoS 2 messages from the client until this QoS 2 message transmission is complete or is discarded due to timeout.</br>
+After this queue is full, the processing of QoS 2 messages will be stopped."""
+            zh: """EMQX 会在客户端的会话中创建一个等待队列来接受客户端的 QoS 2 消息，直到这条 QoS 2 消息传输完成或由于超时被丢弃。</br>
+该队列满后会停止处理 QoS 2 消息。"""
         }
         label: {
             en: """Max Awaiting PUBREL"""
-            zh: """Max Awaiting PUBREL"""
+            zh: """PUBREL 等待队列长度"""
         }
     }
 
     mqtt_await_rel_timeout {
         desc {
-            en: """The QoS 2 messages (Client -> Broker) will be dropped if awaiting PUBREL timeout."""
-            zh: """PUBREL (Client -> Broker) 最大等待时间，超时则会被丢弃。"""
+            en: """The QoS 2 messages (Client -> Broker) handle flow will be ignored if awaiting PUBREL timeout. This QoS 2 message can be published successfully."""
```

Comment:
what do you mean on "can be published successfully?"

Does it violate the [MQTT 4.3.3-10] ?

Reply:
Quoted the MQTT 5.0 - [MQTT 4.3.3 -10]
> Until it has received the corresponding PUBREL packet, the receiver MUST acknowledge any subsequent PUBLISH packet with the same Packet Identifier by sending a PUBREC. It MUST NOT cause duplicate messages to be delivered to any onward recipients in this case

IMO, the [4.3.3 - 10] x is just saying how we should ack a duplicate QoS2 message. Instead of saying when should we post this message to the topic?

I think the violation of [4.3.3 - 10] is implemented in other lines?
https://github.com/emqx/emqx/blob/c01eae5a8ba330b3b4c7f5e4876eab70594e7dae/apps/emqx/src/emqx_session.erl#L392-L393

Reply:
I think we have the right timing for publishing messages to the topic? That has described in [MQTT 5.0 - Figure 4.3](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901238)